server/serverRun.py

```diff
@@ -245,13 +263,15 @@ def create_item():
 @app.route('/logout', methods=['GET', 'POST'])
 def logout():
     session.pop('username', None)
-    return redirect(url_for('login_page'))
+    return redirect(url_for('login_page')), 200
```

Comment:
This is the correct line. Do we need this server_side redirect? Vue does the redirect with <router-link>s on the client-side.

The /logout endpoint has to revoke the cookie or something.

Reply:
Yeah I didn't get around to creating a ticket or fixing this fully, but the logout button doesn't work as it is now. It does the redirect in Vue, but it has to actually make a request to this endpoint and it doesn't. Good point that this doesn't need a redirect. Created #66